modules/programs/broot.nix

```
@@ -194,24 +194,17 @@ in {
           # Dummy file to prevent broot from trying to reinstall itself
           (pkgs.writeTextDir "launcher/installed-v1" "")
         ];
-
         postBuild = ''
-          ln -s ${
-            tomlFormat.generate "broot-config" cfg.settings
-          } $out/conf.toml
-
-          # Remove conf.hjson, whose content has been merged into programs.broot.settings
           rm $out/conf.hjson
+          ${lib.getExe pkgs.jq} --slurp add > $out/conf.hjson \
+            <(${
+              lib.getExe pkgs.hjson-go
+            } -c ${cfg.package.src}/resources/default-conf/conf.hjson) \
+            ${jsonFormat.generate "broot-config.json" cfg.settings}
```

No need for the `rm` if your redirection would overwrite the file anyway.

```suggestion
          ${lib.getExe pkgs.jq} --slurp add > $out/conf.hjson \
            <(${
              lib.getExe pkgs.hjson-go
            } -c ${cfg.package.src}/resources/default-conf/conf.hjson) \
            ${jsonFormat.generate "broot-config.json" cfg.settings}
```

---

`rm` is necessary because `conf.hjson` is a symlink to a read-only file in the nix store, created by `symlinkJoin`, and `>` tries to truncate the target of the symlink instead of removing the link itself.